api/utils/cron.js

```diff
@@ -0,0 +1,9 @@
+const CronJob = require('cron').CronJob;
+const DatabaseBackup = require('../db/backup.js');
+
+// Configure cron job to run every day at 12:00am
+let job = new CronJob('00 00 00 * * *', function () {
```

Comment:
Can this be a `const`, or is it required to be `let`? Also, we use arrow functions everywhere, so it would be better to use one for the callback too.